Import/Esri/ArcGISRuntime/Toolkit/Controls/CppApi/CoordinateConversion.qml

```diff
@@ -0,0 +1,689 @@
+/*******************************************************************************
```

Comment:
I don't think any of these `Import/*` qml files should be included - these are from the old pattern